stratum/hal/lib/common/utils.h

```diff
@@ -231,11 +232,11 @@ std::string ConvertMediaTypeToString(const MediaType& type);
 std::string ConvertHwStateToPresentString(const HwState& hw_state);
 
 // Converts ::gnmi::Decimal64 to float type.
-::util::StatusOr<float> Decimal64ValueToFloat(const ::gnmi::Decimal64& value);
+::util::StatusOr<double> Decimal64ValueToDouble(const ::gnmi::Decimal64& value);
```

Comment:
Should this be named `ConvertDecimal64ToDouble` for consistency with other utlils functions?

Reply:
yes, `ConvertDecimal64ToDouble` is better